### PR TITLE
feat: merge quotes and reviews,

### PR DESCRIPTION
**Pull request by rosejbon**
This Trello ticket: https://trello.com/c/kSpOyjOl/137-ability-to-add-multiple-what-other-people-have-said-and-reviews-vv-e

Merges quotations and reviews, so we can have multiple quotes from young people. Transferred all existing quotes to reviews.

### CODE REVIEW

**Comment by marxian** [comment]
#### 📝Changed routes:


| [**/service/\[id\]**](https://my-best-life-nvzphl7cp-neontribe.vercel.app/service/[id]) |
|:-:|
|<a href="https://my-best-life-nvzphl7cp-neontribe.vercel.app/service/[id]">Screenshot of /service/[id]</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/SbCbymAO7UxP4lPD6NS1TOctzNH_J3UmlqqRRcz6Qwm_I8JHNkW4VFR9R7lpMuKyJQvKHYYbQBvTtc2RY8l-7A.png">(view full size)</a> |







Commit a334d7e7499df7d1a746611602ea2ed1f6f3fd03 (https://my-best-life-nvzphl7cp-neontribe.vercel.app).

**Review by edpe — APPROVED** [review]

File: content/services/brook-dash-lambeth-1.md [hunk 1]
```diff
@@ -30,9 +30,6 @@ area:
   - Gipsy Hill
 time: "Walk-in sessions. The team generally work Mondays to Fridays, but you can
   leave a message in the evening and over weekends. "
-quotation: I felt comfortable to discuss any issues surrounding my health. I was
```
Comment (Nikomus) [inline content/services/brook-dash-lambeth-1.md:33]: This one to be moved into a review?

File: content/services/girls-only-football-coaching-6-a-side.md [hunk 2]
```diff
@@ -28,10 +28,10 @@ makeMapLink: true
 area:
   - Ferndale
 time: Saturdays 10am to 12pm (except summer holiday period)
-quotation: “Coming to St. Matthew’s has really helped me as a person because
+reviews: [{comment: "Coming to St. Matthew’s has really helped me as a person because
   playing here is like a dream. In Madrid, I used to play football, but on the
   streets, so never really in a team. Here, I am playing with other girls, as a
-  team. It’s really nice. I love it.” Kelfra, aged 14."
```
Comment (Nikomus) [inline content/services/girls-only-football-coaching-6-a-side.md:34]: Do we want to move the name into the author part of the review?

File: content/services/reaching-higher-leaders-in-training-lits.md [hunk 3]
```diff
@@ -31,12 +31,6 @@ format: Meeting a group of people
 location: South Norwood Methodist Church, Suffolk Road, London SE25 6EG
 makeMapLink: true
 time: "Tuesday evenings, 6.30pm - 8.30pm. A one-year programme. "
-quotation: The LIT programme I believe has been paramount to making me stand out
```
Comment (Nikomus) [inline content/services/reaching-higher-leaders-in-training-lits.md:34]: This one to move to a review also?

File: content/services/brook-dash-lambeth-1.md (hunk 1 above)
Comment (rosejbon) [reply content/services/brook-dash-lambeth-1.md:33]: Moved :)

File: content/services/reaching-higher-leaders-in-training-lits.md (hunk 3 above)
Comment (rosejbon) [reply content/services/reaching-higher-leaders-in-training-lits.md:34]: 👍🏻 done

File: content/services/girls-only-football-coaching-6-a-side.md (hunk 2 above)
Comment (rosejbon) [reply content/services/girls-only-football-coaching-6-a-side.md:34]: I will do that in the CMS later I think

**Pull request merged**